classes/models/coursework.php

```diff
@@ -819,15 +823,15 @@ public function get_allocatable_identifier_hash($allocatable) {
      * @return plagiarism_base[]
      */
     public function get_plagiarism_helpers() {
-        $enabled_plagiarism_plugins = array_keys(\core_component::get_plugin_list('plagiarism'));
+        $enabledplagiarismplugins = array_keys(\core_component::get_plugin_list('plagiarism'));
         $objects = [];
-        foreach ($enabled_plagiarism_plugins as $plugin_name) {
-            $class_name = "\\mod_coursework\\plagiarism_helpers\\{$plugin_name}";
-            if (class_exists($class_name)) {
+        foreach ($enabledplagiarismplugins as $pluginname) {
+            $classname = "\\modcoursework\\plagiarismhelpers\\{$pluginname}";
```

Comment:
I think some of those _ are meant to stay e.g for mod_coursework but also palgiarism_helpers

Reply:
Reinstated in commit c1ee01c